src/app/public/components/about/about.component.html

```diff
@@ -1,84 +1,198 @@
+						MINT.OBadges ist ein praxisbezogenes Forschungsprojekt. Es zielt darauf ab, Open Badges in
+						außerschulischen Kursangeboten mit MINT-Schwerpunkt erfolgreich zu etablieren.
+						Bildungsinitiativen sollen befähigt werden, den Kompetenzerwerb durch ihre Angebote
+						niedrigschwellig und anschlussfähig zu dokumentieren. Dafür wird der technische Standard um eine
+						inhaltliche Komponente (ESCO Standard) ergänzt. Die Entwicklung eines KI-Badge-Assistenten
+						erleichtert den Initiativen die Klassifizierung und Vergabe. Durch die Erprobung von zwei
+						digitalen und drei analogen Reallaboren wird die Integration von Open Badges erforscht und
+						entsprechende Communities aktiviert.  Das Forschungsprojekt MINT.OBadges wird von der matrix
+						gGmbH, der mycelia gGmbH und der openSenseLab gGmbH als Verbund gemeinsam umgesetzt und an Mein
+						Bildungsraum angeschlossen. Gefördert wird das Projekt vom Bundesministerium für Bildung und
+						Forschung.
+					</p>
+				</div>
+			</div>
+
+			<div style="margin-right: 5em">
+				<div class="projects-logo-style">
+					<img src="../assets/logos/LogoKlimaCampus.png" alt="Logo for MintOBadges" />
+				</div>
+				<div class="projects-desc-style">
+					<h3>Klima Campus</h3>
+					<p class="text">
+						Der Klima Campus ist ein transformativer Lernraum, der junge Menschen zum Thema natürlicher
+						Klimaschutz informiert und zum Handeln motiviert. Ziel ist es, die Akzeptanz des Natürlichen
+						Klimaschutzes in der gesamten Gesellschaft zu verbreiten und so mittel- und langfristig den
+						Ausstoß von Treibhausgasen (THG) durch die Änderung eigener Verhaltensweisen, wie auch durch
+						strukturelle Veränderungen, zu verringern. Durch den Klima Campus und die daran angeschlossenen
+						Aktivitäten werden Kompetenzen erworben - diese werden durch Open Educational Badges
+						dokumentiert und für die Lernenden transparent. Der Klima Campus ist ein Projekt der mycelia
+						gGmbH und wird vom Bundesministerium für Umwelt, Naturschutz, nukleare Sicherheit und
+						Verbraucherschutz gefördert. 
+					</p>
 				</div>
 			</div>
 		</div>
 	</div>
 
-	<div class="formsection formsection-nohelp u-margin-bottom4x">
-		<h2 class="formsection-x-title">{{ 'About.funding' | translate }}</h2>
+	<!-- Funded by -->
+	<div class="fund-style">
+		<div>
+			<img src="../assets/logos/Logo-Telekom-Stiftung.png" alt="Logo for BMBF" />
+		</div>
 
-		<div class="formsection-x-body">
-			<div class="foerderung">
-				<img src="assets/logos/BMBF_CMYK_Gef_L.png" />
-				<div>
-					<p>
-						{{ 'About.fundingDesc' | translate }}
-					</p>
-					<a
-						href="https://www.photonikforschung.de/projekte/open-innovation/foerdermassnahme/open-photonik-pro.html"
-						>{{ 'About.moreAboutFund' | translate }}</a
-					>
+		<div>
+			<img src="../assets/logos/Logo-BMBF.png" alt="Logo for BMBF" />
+		</div>
+
+		<div><img src="../assets/logos/Logo-BMUV.png" alt="Logo for BMBF" /></div>
```

Comment:
This alt text doesn't seem to fit here

Reply:
Thanks timber for the review! This PR still in progress, I just created a pull request to send use the preview.